content/docs/command-reference/plots/diff.md

```diff
@@ -6,76 +6,77 @@ plotting them in a single image.
 ## Synopsis
 
 ```usage
-usage: dvc plots diff [-h] [-q | -v] [-t [TEMPLATE]] [-d [DATAFILE]] [-f FILE]
-                     [-s SELECT] [-x X] [-y Y] [--stdout] [--no-csv-header]
-                     [--no-html] [--title TITLE] [--xlab XLAB] [--ylab YLAB]
-                     [revisions [revisions ...]]
+usage: dvc plots diff [-h] [-q | -v] [-t <name_or_path>]
+                      [--targets [<path> [<path> ...]]] [-o <path>]
+                      [-x <field>] [-y <field>] [--no-csv-header]
+                      [--show-vega] [--title <text>]
+                      [--x-label <text>] [--y-label <text>]
+                      [revisions [revisions ...]]
 
 positional arguments:
-  revisions             Git commits to plot from
+  revisions             Git commits to plot from/to
```

Comment:
what does `/to` part mean?

Reply:
That you can chose the from revision and also the to revision(s).

Reply:
hmm ... feels really weird

why just "Git commits to read metrics from to plot" is not enough?

Reply:
I went for `Git commits to find metrics to diff` for now (in d22c9c65). Cc @efiop

Reply:
UPDATE: Actually I changed `diff` for `compare` in edd0323c